incubator/jaeger/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.9.0
 description: A Jaeger Helm chart for Kubernetes
 name: jaeger
-version: 0.8.3
+version: 0.9.0
```

Comment:
Looks like a breaking change, you should bump a major version and document how to upgrade according to documentation: https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#versioning

Reply:
True, I figured not that many people must be using this chart  but the Upgrading section has been added to README.
Please review!

Reply:
I'd like to save `1.0.0` for when this goes to `stable` repo however